app/i18n/index.js

```diff
@@ -12,9 +12,10 @@ const languages = {
     hu: { translation: require('./lang/hu.json') },
     it: { translation: require('./lang/it.json') },
     nl: { translation: require('./lang/nl.json') },
-    pt: { translation: require('./lang/pt-br.json') },
+    ptBR: { translation: require('./lang/ptBR.json') },
     ru: { translation: require('./lang/ru.json') },
-    sq: { translation: require('./lang/sq.json') }
+    sq: { translation: require('./lang/sq.json') },
+    zhCN: { translation: require('./lang/zhCN.json') }
```

Comment:
zhCN is not a proper language code, please use zh-CN as this is compared with the return value of navigator.language, see https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language

Reply:
Fixed